src/main/resources/org/jenkinsci/plugins/workflow/support/steps/input/Messages.properties

```diff
@@ -0,0 +1,8 @@
+paused_for_input=Paused for Input
+pipeline_need_input=Pipeline has paused and needs your input before proceeding
+wait_for_interactive_input=Wait for interactive input
+Abort=Abort
+Rejected=Rejected
+Rejected_by=Rejected by {0}
```

Comment:
Looks like the build failure is because Findbugs doesn't like that the generated methods for these 3 keys are capitalized.